### ISSUE TITLE
commands get called multiple times

### ISSUE THREAD

**Issue by dakyskye** [dakyskye/dxhd#17]
sometimes I just start dxhd, it starts without any error, but when I trigger a keybinding, dxhd executes commands two or three times!

**Comment by dakyskye** [comment]
I am not sure, but I sort of fixed it probably: 
![image](https://user-images.githubusercontent.com/32128756/85305398-0fcb7780-b4be-11ea-91d8-f7f35d54df1f.png)

Also, now dxhd **always** reports when a command fails, unlike to then! then dxhd reported it only the first time command failed!

**Comment by dakyskye** [comment]
Worst thing about this bug is that I can't replicate it, I have no idea when it happens, and it happens barely! However, I just managed to reproduce it here, in the screenshot!

**Comment by dakyskye** [comment]
Should be fixed by latest commits.

**Comment by dakyskye** [comment]
```sh
⇒  dxhd -v
you are using dxhd, version master-6212aaf
``` apparently latest commits did not fix that! That's so sad, but at least we know it is not fixed.

**Comment by dakyskye** [comment]
Hm it is so weird, I triggered a binding that opens my terminal, and it did open my terminal twice.  Then, I opened this issue, thus I just ran `dxhd -v`.  I did not restart/kill dxhd, and it works properly now.  That is very weird.

**Comment by dakyskye** [comment]
Yup here we go, I started `dxhd` like 50 times with DEBUG variable set.  It was feeling good, until it did execute command twice :(
![image](https://user-images.githubusercontent.com/32128756/85308652-81a5c000-b4c2-11ea-9357-898138bfc033.png)

```json
{"level":"debug","ts":"2020-06-22T19:55:35.044+0400","caller":"dxhd-git/listener.go:59","msg":"now executing a command","command":"alacritty"}
{"level":"debug","ts":"2020-06-22T19:55:35.045+0400","caller":"dxhd-git/listener.go:59","msg":"now executing a command","command":"alacritty"}
``` these lines, we see it executed the same command for some reason the same time! First at `2020-06-22T19:55:35.044+0400`, second at `2020-06-22T19:55:35.045+0400`

**Comment by dakyskye** [comment]
That seems to be fixed by the latest commit! f7f5df8
